analysis_options.yaml

```diff
@@ -1,7 +1,7 @@
 include: package:pedantic/analysis_options.yaml
 analyzer:
-  language:
-    strict-casts: true
+  strong-mode:
```

Comment:
I assume the previous version is the one we want?

Reply:
ah yeah, the woes of having commits on both sides. Fixed.